### ISSUE TITLE
What is the data format?

### ISSUE THREAD

**Issue by ltwltw** [AmirooR/IntraOrderPreservingCalibration#1]
Can you explain what the data format under data looks like?

**Comment by AmirooR** [comment]
Hi, most of the logits/label pairs are taken from [this repo](https://github.com/markus93/NN_calibration) as mentioned in [footnote 5 of the paper](https://arxiv.org/pdf/2003.06820.pdf). I will add a link for the remaining datasets soon.  Please take a look at [here](https://github.com/AmirooR/IntraOrderPreservingCalibration/blob/main/builder.py#L56) for the data format.
